content/ja/docs/tasks/configure-pod-container/assign-pods-nodes.md

```diff
@@ -7,13 +7,13 @@ weight: 120
 <!-- overview -->
 このページでは、KubernetesのPodをKubernetesクラスター上の特定のノードに割り当てる方法を説明します。
 
-## {{% heading "prerequisites" %}}
+## {{% heading "prerequisites" %}} {#before-you-begin}
```

Comment:
Currently, the anchor of heading shortcode  will be specified like this. I think It is currect.
In the future, I think it would be good to generate the anchor in the headering shortcode.